autopush/db.py

```diff
@@ -266,6 +266,9 @@ def wrapper(self, *args, **kwargs):
         except ProvisionedThroughputExceededException:
             self.metrics.increment("error.provisioned.%s" % func.__name__)
             raise
+        except JSONResponseError:
+            self.metrics.increment("error.jsonresponse.%s" % func.__name__)
+            raise
         except BotoServerError:
```

Comment:
if we're willing to change these metric names the above 2 can collapse into this w/ increment("error.%s.%s" % (type(e).__name__, func.__name__)